website/projects/scimaps/src/app/shared/components/profile-gallery/profile-gallery.component.html

```diff
@@ -3,7 +3,7 @@ <h2 class="title" *ngIf="title !== ''">{{ title }}</h2>
 
 <div class="profile-container">
   <div class="profile" *ngFor="let profile of profiles" (click)="goToLink(profile.link)">
-    <img class="profile-image" [src]="getImageSource(profile)" [alt]="profile.slug + '/image.png'">
+    <img class="profile-image" [src]="getImageSource(profile)" [alt]="profile.slug + '/'+ profile.image">
```

Comment:
Move the creation of the 'real image link' to the resolver. Then profile.image will be the actual (relative) url to use.